### ISSUE TITLE
Form Request must extends FormRequest class for validation.

### ISSUE THREAD

**Issue by pankitgami** [caffeinated/modules#253]
php artisan make:module:request CLI command create a request class for validators, but it is extending App\Http\Requests\Request class in your stub of a request.

Following is the code of request.stub:

```
<?php

namespace DummyNamespace;

use DummyRootNamespaceHttp\Requests\Request; (Change this to FormRequest as shown below. )

class DummyClass extends Request
{
     /**
      * Determine if the user is authorized to make this request.
      *
      * @return bool
      */
     public function authorize()
    {
        return false;
    }

    /**
     * Get the validation rules that apply to the request.
     *
     * @return array
     */
    public function rules()
    {
        return [
            //
         ];
    }
}
```

This is the code of laravel generated request class.

```
<?php

namespace App\Http\Requests;

use Illuminate\Foundation\Http\FormRequest;

class TestRequest extends FormRequest
{
    /**
     * Determine if the user is authorized to make this request.
     *
     * @return bool
     */
    public function authorize()
    {
        return false;
    }

    /**
     * Get the validation rules that apply to the request.
     *
     * @return array
     */
    public function rules()
    {
        return [
            //
        ];
    }
}
```

**Comment by A-Lawrence** [comment]
This was just resolved in #265.  @kaidesu any chance of closing, please?

**Comment by kaidesu** [comment]
Yup!
